recipes/configs/phi3/mini_full_low_memory.yaml

```diff
@@ -43,13 +43,13 @@ checkpointer:
     model-00002-of-00002.safetensors
   ]
   recipe_checkpoint: null
-  output_dir: /tmp/Phi-3-mini-4k-instruct
+  output_dir: /tmp/finetuned-phi3-4k-model
   model_type: PHI3_MINI
 resume_from_checkpoint: False
 
 # Fine-tuning arguments
-batch_size: 2
-epochs: 3
+batch_size: 16
```

Comment:
Made a large batch size for performance reasons

Reply:
How does this BS impact memory utilization?

Reply:
So I ended up decreasing bs to 4 in order to fit < 24 GB. There is a memory spike that occurs for batches that contain sequences up to max_seq_len of 4096. This alludes to a bigger problem we face in handling large seq lengths. Chunked cross entropy reliefs a little pressure, but we should keep digging here. 

cc @rohan-varma  and @ebsmothers who we're the ones that debugged this issue.